Program.cs

```diff
@@ -1,143 +1,86 @@
-﻿// Welcome to AssemblyAI! Get started with the API by transcribing
-// a file using C#.
-//
-// In this example, we'll transcribe a local file. Get started by
-// downloading the snippet, then update the 'filename' variable
-// to point to the local path of the file you want to upload and
-// use the API to transcribe.
-//
-// IMPORTANT: Update line 130 to point to a local file.
-//
-// Have fun!
-
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
 using System.Text;
-using System.Threading;
+using System.Text.Json;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
```

Comment:
Newtonsoft.Json is great 3rd party library that filled a gap in the built-in .NET APIs. This gap has since been filled by System.Text.Json, so it's better to use those APIs which are more performant and don't require an external dependency, also less steps since you don't have to add the Newtonsoft JSON nuget package.